src/commands/edeliver-deploy.yml

```diff
@@ -0,0 +1,29 @@
+# TODO: Split this into another orb (elixir-deploy) ?
```

Comment:
should we create issues for these todo's so that the circleci team and community could contribute? Like an open discussion

Reply:
I will be waiting for CircleCI team feedback first.